src/applications/disability-benefits/all-claims/pages/index.js

```diff
@@ -104,6 +104,7 @@ export {
   ancillaryFormsWizardSummary,
   choosePtsdType,
   claimExamsInfo,
+  claimType,
```

Comment:
missing bit of alphabetical cleanup, @cvalarida 😉

Reply:
Weeeeeird. I thought we already cleaned this up. :man_shrugging:

Reply:
Aha! We _did_ already do this! https://github.com/department-of-veterans-affairs/vets-website/pull/9930/files#diff-e3260aa845f2b3d5d40e8ea9cea0e6d4R63

Looks like it's not causing any merge conflicts, though, so that's nice. :smile:

Reply:
¯\_(ツ)_/¯ mumble mumble git stuff